lib/services/nativescript-migration-service.ts

```diff
@@ -4,16 +4,19 @@
 import Future = require("fibers/future");
 import path = require("path");
 import helpers = require("./../helpers");
+import semver = require("semver");
 
 export class NativeScriptMigrationService implements IFrameworkMigrationService {
 	private static TYPESCRIPT_ABBREVIATION = "TS";
 	private static JAVASCRIPT_ABBREVIATION = "JS";
 	private static SUPPORTED_LANGUAGES = [NativeScriptMigrationService.JAVASCRIPT_ABBREVIATION, NativeScriptMigrationService.TYPESCRIPT_ABBREVIATION];
 	private static REMOTE_NATIVESCRIPT_MIGRATION_DATA_FILENAME = "NativeScript.json";
+	private static PACKAGE_JSON_FILE_NAME = "package.json";
 	private nativeScriptResourcesDir: string;
 	private nativeScriptMigrationFile: string;
 	private tnsModulesDirectoryPath: string;
 	private remoteNativeScriptResourcesPath: string;
+	private nativeScriptDefaultPackageJsonFile: string;
```

Comment:
Also we're inside nativescript-migration-service, do we need `nativescript` prefix?